// remove-stopwords/lib/export_file.js
export const all = require('./languages_all.js').languages
export const af = require('./stopwords_af.js').words
export const ar = require('./stopwords_ar.js').words
export const bg = require('./stopwords_bg.js').words
export const bn = require('./stopwords_bn.js').words
export const br = require('./stopwords_br.js').words
export const ca = require('./stopwords_ca.js').words
export const cs = require('./stopwords_cs.js').words
export const da = require('./stopwords_da.js').words
export const de = require('./stopwords_de.js').words
export const el = require('./stopwords_el.js').words
export const en = require('./stopwords_en.js').words
export const eo = require('./stopwords_eo.js').words
export const es = require('./stopwords_es.js').words
export const et = require('./stopwords_et.js').words
export const eu = require('./stopwords_eu.js').words
export const fa = require('./stopwords_fa.js').words
export const fi = require('./stopwords_fi.js').words
export const fr = require('./stopwords_fr.js').words
export const ga = require('./stopwords_ga.js').words
export const gl = require('./stopwords_gl.js').words
export const ha = require('./stopwords_ha.js').words
export const he = require('./stopwords_he.js').words
export const hi = require('./stopwords_hi.js').words
export const hr = require('./stopwords_hr.js').words
export const hu = require('./stopwords_hu.js').words
export const hy = require('./stopwords_hy.js').words
export const js = require('./stopwords_id.js').words
export const it = require('./stopwords_it.js').words
export const ja = require('./stopwords_ja.js').words
export const ko = require('./stopwords_ko.js').words
export const la = require('./stopwords_la.js').words
export const lv = require('./stopwords_lv.js').words
export const mr = require('./stopwords_mr.js').words
export const nl = require('./stopwords_nl.js').words
export const no = require('./stopwords_no.js').words
export const pl = require('./stopwords_pl.js').words
export const pt = require('./stopwords_pt.js').words
export const ro = require('./stopwords_ro.js').words
export const ru = require('./stopwords_ru.js').words
export const sk = require('./stopwords_sk.js').words
export const sl = require('./stopwords_sl.js').words
export const so = require('./stopwords_so.js').words
export const st = require('./stopwords_st.js').words
export const sv = require('./stopwords_sv.js').words
export const sw = require('./stopwords_sw.js').words
export const th = require('./stopwords_th.js').words
export const tr = require('./stopwords_tr.js').words
export const yo = require('./stopwords_yo.js').words
export const zh = require('./stopwords_zh.js').words
export const zu = require('./stopwords_zu.js').words
